src/modules/timeAgo/index.ts

```diff
@@ -0,0 +1,110 @@
+import { digitsFaToEn } from "../digits";
+
+/**
+ * Convert a date-time value to timestamp.
+ *
+ * @param {string} datetime Format must be yyyy/mm/dd hh:mm:ss.
+ * @returns {number} Calculates the timestamp of the input.
+ */
+export function convertToTimeStamp(datetime: string): number {
+	const patternDateTime = /(\d+)\/(\d+)\/(\d+) (\d+):(\d+):(\d+)/;
+	const dateTime = datetime.match(patternDateTime);
+	if (dateTime) {
+		const date: Date = new Date(
+			Number(dateTime[1]),
+			parseInt(dateTime[2], 10) - 1,
+			Number(dateTime[3]),
+			Number(dateTime[4]),
+			Number(dateTime[5]),
+			Number(dateTime[6]),
+		);
+		return date.getTime();
+	} else throw new TypeError("PersianTools: convertToTimeStamp - The input format must be yyyy/mm/dd hh:mm:ss");
+}
+
+/**
+ * Get current timestamp of current date-time.
+ *
+ * @returns {number} Calculates the timestamp of current date-time.
+ */
+export function getTimeNow(): number {
+	const currentDateTime: string = new Date(Date.now())
+		.toLocaleString("fa-IR", {
+			year: "numeric",
+			month: "2-digit",
+			day: "2-digit",
+			hour: "2-digit",
+			minute: "2-digit",
+			second: "2-digit",
+		})
+		.replace(/‏|،/g, "");
+
+	return convertToTimeStamp(digitsFaToEn(currentDateTime));
+}
+
+/**
+ * Check format of Input.
+ *
+ * @param {string} datetime Format must be yyyy/mm/dd hh:mm:ss.
+ * @returns {boolean} If format of datetime is ok, return true.
+ */
+export function checkFormatDateTime(datetime: string): boolean {
+	return Boolean(datetime.match(/^\d{4}\/\d{1,2}\/\d{1,2} \d{1,2}:\d{1,2}:\d{1,2}$/));
+}
+
+/**
+ * Converting Jalali date-time into a time ago.
+ *
+ * @category timeAgo
+ *
+ * @param {string} datetime Format must be yyyy/mm/dd hh:mm:ss. If no value is entered for input, the current time is considered.
+ * @returns {string} Return time ago value. Eg: حدود 1 سال قبل
+ */
+export default function timeAgo(datetime = ""): string {
+	if (typeof datetime !== "string") throw new TypeError("PersianTools: timeAgo - The input must be string");
+
+	if (!checkFormatDateTime(datetime) && datetime !== "")
+		throw new TypeError("PersianTools: timeAgo - The input format must be yyyy/mm/dd hh:mm:ss");
+
+	// Timestamp DateTime
+	let tsDateTime: number;
+
+	// If input be null then tsDateTime get current timestamp
+	if (datetime === "") tsDateTime = getTimeNow();
+	else tsDateTime = convertToTimeStamp(datetime);
+
+	// TimeNow
+	const tsTimeNow: number = getTimeNow();
+
+	// TimeAgo
+	const minute: number = 60 * 1000,
```

Comment:
There is no need to write `number` type while your value is a primitive type. Because typescript will assign the number as variable type automatically in this case.

Reply:
Thanks @ali-master. You learn me a lot 🍀.